python/example_code/cognito/scenario_signup_user_with_mfa.py

```diff
@@ -0,0 +1,164 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows you how to use the AWS SDK for Python (Boto3) with Amazon Cognito to
+do the following:
+
+1. Sign up a user with a user name, password, and email address.
+2. Confirm the user from a code sent in email.
+3. Set up multi-factor authentication by associating an MFA application with the user.
+4. Sign in by using a password and an MFA code.
+5. Register an MFA device to be tracked by Amazon Cognito.
+6. Sign in by using a password and information from the tracked device. This avoids the
+   need to enter a new MFA code.
+
+This scenario requires an existing Amazon Cognito user pool that is configured to
+allow self sign-up, and a client ID that can be used to authenticate with Amazon
+Cognito.
```

Comment:
This scenario requires the following resources: 
* An existing Amazon Cognito user pool that is configured to allow self sign-up
* A client ID to use for authenticating with Amazon Cognito

Reply:
fixed